flink-python/pyflink/datastream/stream_execution_environment.py

```diff
@@ -553,6 +553,29 @@ def set_python_executable(self, python_exec: str):
             .getEnvironmentConfig(self._j_stream_execution_environment)
         env_config.setString(jvm.PythonOptions.PYTHON_EXECUTABLE.key(), python_exec)
 
+    def add_jars(self, key: str, jars_path: str):
```

Comment:
Why not add two methods that `add_jars` for `pipeline.jars` and `add_class_paths` for `pipeline.classpaths`

Reply:
Yes, it will be more intuitive if we separate it into two interfaces named add_jars() and add_classpaths().